File: 프로그래머스/1/135808. 과일 장수/과일 장수.py
def solution(k, m, score):
    answer = 0
    score.sort(reverse = True) # 과일을 오름차순으로 정렬
    
    # 오름차순으로 정렬된 과일의 점수를 한박스당 가장 낮은 점수일 첫번째 박스의 m번째부터 m개 단위로 순회하면서 
    for i in range(m-1, len(score), m):
        answer += score[i] * m # 값을 answer에 담아준다
    return answer
    

File: 프로그래머스/1/135808. 과일 장수/과일 장수.js
function solution(k, m, score) {
    var answer = 0;
    score.sort((a,b) => (b-a)); //배열을 내림차순으로 정렬 
    
    // 내림차순 정렬된 배열에서 가장 작은수의 인덱스를 갯수단위로 순회하면서
    for (let i = m-1; i < score.length; i += m) {
        answer += score[i] * m; // 해당 사과의 가격을 한박스당 숫자에 곱하여 계산
    }
    
    return answer;
}